netcheck.sh

```diff
@@ -94,7 +96,7 @@ PRINT_DISCONNECTED() {
 DISCONNECTED_EVENT_HOOK() {
   if [[ $VAR_ACT_ON_DISCONNECT = true ]]; then :
     COMMAND="$VAR_DISCONNECT_SCRIPT &"
-    echo -e $COLOR_RED"$STRING_2 EXEC $COMMAND"$COLOR_RESET
+    echo -e $COLOR_CYAN"$STRING_2 EXEC $COMMAND"$COLOR_RESET
```

Comment:
I think using red here is an appropriate color to indicate a down link and provides the maximum contrast to the nominal state of green. But I'm open to discussion if you have a strong opinions. I'd love to better understand your thought behind the change.

Reply:
Maybe for the disconnected event red makes sense (I would not care), but I was confused about the reconnect event hook exec being in red when scrolling the console output. At first I considered changing the reconnect exec line to green, but at the end decided to change both exec events to a different color to have a single green line for reconnect event and a single red line for disconnect event when looking at the output. My reasoning is that executing an external script is neither a positive nor a negative event.

Reply:
Thanks for the explanation.  I would vote we leave disconnected as red (negative event), reconnect green (positive), to mirror the functionality when not executing a script to maintain familiarity to the user, but if you'd like you could break out the execution string to be a different (or no) color.

Reply:
That is exactly what I did (at least what I have tried to do :-)). I have kept the colors for the disconnect/reconnect events and used separate color (cyan) only for the external script exec lines.

Reply:
In your changes you are still including $STRING_2 (LINK DOWN:) as cyan, but honestly I think this is not a big deal, lets just close this out! I really appreciate your contributions to this project. I hope its being useful for you.